### PR TITLE
Small changes. This is a lousy commit message.

### PR DESCRIPTION
**Pull request by stianlagstad**

### CODE REVIEW

File: src/Main.elm [hunk 1]
```diff
@@ -270,10 +289,10 @@ viewSpeaker speaker =
                 , td [] [ text group ]
                 ]
     in
-    row speaker.id speaker.name speaker.group False
-        :: (speaker.replies
-                |> List.map (\r -> row r.id r.name r.group True)
-           )
+        row speaker.id speaker.name speaker.group False
```
Comment (runarfu) [inline src/Main.elm]: Do we use different versions of `elm-format`? I use `elm-format-0.18 0.7.0-exp`.

**Review by runarfu — APPROVED** [review]
:shipit: but we should make sure we use `elm-format` in the same way, to avoid all the whitespace diffs.

File: src/Main.elm (hunk 1 above)
Comment (stianlagstad) [reply src/Main.elm]:
```
➜  ~ elm-format --version
Invalid option `--version'

elm-format-0.18 0.6.1-alpha
```
Should I update the version and see if anything changes?

Comment (runarfu) [reply src/Main.elm]: Please do :)

Comment (stianlagstad) [reply src/Main.elm]: Am I using an old npm or something? I keep getting that elm-format version. Which npm version do you have?

```
➜  elm-frontend git:(ch-ch-ch-ch-changes) npm --version                                                  
5.6.0                     
➜  elm-frontend git:(ch-ch-ch-ch-changes) npm install -g elm-format                                      
/home/stian/.nvm/versions/node/v9.5.0/bin/elm-format -> /home/stian/.nvm/versions/node/v9.5.0/lib/node_modules/elm-format/bin/elm-format

> elm-format@0.6.1-alpha install /home/stian/.nvm/versions/node/v9.5.0/lib/node_modules/elm-format       
> binwrap-install         

+ elm-format@0.6.1-alpha  
updated 1 package in 5.546s                         
➜  elm-frontend git:(ch-ch-ch-ch-changes) elm-format --version                                           
Invalid option `--version'

elm-format-0.18 0.6.1-alpha
```

Comment (runarfu) [reply src/Main.elm]: Aha, this is my bad, I've installed an experimental version of `elm format` and forgot about it. Read about it here: https://github.com/avh4/elm-format/blob/master/Release%20Notes/0.7.0-exp.md.
I like it however, what do you think?

Comment (stianlagstad) [reply src/Main.elm]: What do you think? https://github.com/roisalen/elm-frontend/pull/4/commits/69ca77a7539f046551230b26302162b3fff3234d

I now have elm-format-0.18 0.7.0-exp.

**Review by runarfu — APPROVED** [review]

**Pull request merged**